README.md

```diff
@@ -446,6 +446,9 @@ When deserializing, the otherwise object-ignorant data will be upcast back to `P
 
 `arrayType` works the exact same way on a `DictionaryField`.
 
+When using `ScalarType` then all values have to conform to the single type, no exceptions are allowed not even `null` values. Any divergence will cause an exception to be thrown.
+Also beware that if the property is set to not match `arrayType` then this will be also serialized out!
```

Comment:
huh?

Reply:
I guess that the problem is with
> Also beware that if the property is set to not match `arrayType` then this will be also serialized out!

This means that if you have
```
#[Sequence(arrayType: ScalarType::STRING)]
public array $listOfStrings
```

And you assign into the `listOfStrings` the values `[1, 2, "3"]` and serialize it then it will be serialized... 

In other words no output type check, only on deserialization.

Reply:
I had a hard time following the whole thing. 😄